src/server/sched_func.c

```diff
@@ -981,10 +957,21 @@ set_sched_default(pbs_sched *psched, int from_scheduler)
 		psched->sch_attr[SCHED_ATR_throughput_mode].at_flags |= ATR_SET_MOD_MCACHE | ATR_VFLAG_DEFLT;
 	}
 
+	if (!(psched->sch_attr[SCHED_ATR_version].at_flags & ATR_VFLAG_SET)) {
+		set_attr_generic(&(psched->sch_attr[SCHED_ATR_version]), &sched_attr_def[SCHED_ATR_version], PBS_VERSION, NULL, SET);
```

Comment:
If we are just going to set this to the server's version, remove the attribute.  You can get it from the server itself.

Reply:
I agree with @bhroam. We can just remove this attribute for Scheduler.

Reply:
@bhroam You only told in one of our mail discussion that there is someone who is using this attribute, if that's not true then sure I will remove this attribute.

Reply:
I'm sorry if I gave that impression.  I thought I said I didn't know if anyone was using the attribute.  In any case, right now it is not the scheduler's version, it is the server's version.  If for any reason someone did start up a scheduler with a different version, we'd print the wrong one.  So either remove the attribute, or have the scheduler set it.  I say remove it.  You might want to check with the PMs first though.